client/src/partials/templates/employeeList.tmp.html

```diff
@@ -0,0 +1,4 @@
+<a>
+  <span bind-html-unsafe="match.id | typeaheadHighlight:query"></span>
+  <span>{{ match.model.prenom + '  ' + match.model.name + '  ' + match.model.postnom + ' [' + match.model.code_employee + ']'}}</span>
```

Comment:
I think you've now included all this information in `employee.label`.  Correct me if I am wrong, but I think can be
```html
<span> {{ match.model.label }} </span>
```
which is  much more efficient.